modules/registry-refactoring/src/main/java/org/apache/airavata/registry/core/entities/expcatalog/ProcessEntity.java

```diff
@@ -22,40 +22,109 @@
 
 
 import javax.persistence.*;
+import java.io.Serializable;
+import java.sql.Timestamp;
 import java.util.List;
 
+/**
+ * The persistent class for the process database table.
+ */
 @Entity
-@Table(name = "EXPCAT_PROCESS")
-public class ProcessEntity {
+@Table(name = "PROCESS")
+public class ProcessEntity implements Serializable {
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @Column(name = "PROCESS_ID")
     private String processId;
+
+    @Column(name = "EXPERIMENT_ID")
     private String experimentId;
-    private long creationTime;
-    private long lastUpdateTime;
+
+    @Column(name = "CREATION_TIME")
+    private Timestamp creationTime;
+
+    @Column(name = "LAST_UPDATE_TIME")
+    private Timestamp lastUpdateTime;
+
+    @Lob
+    @Column(name = "PROCESS_DETAIL")
     private String processDetail;
+
+    @Column(name = "APPLICATION_INTERFACE_ID")
     private String applicationInterfaceId;
+
+    @Column(name = "APPLICATION_DEPLOYMENT_ID")
     private String applicationDeploymentId;
+
+    @Column(name = "COMPUTE_RESOURCE_ID")
     private String computeResourceId;
+
+    @Lob
+    @Column(name = "TASK_DAG")
     private String taskDag;
+
+    @Column(name = "GATEWAY_EXECUTION_ID")
     private String gatewayExecutionId;
+
+    @Column(name = "ENABLE_EMAIL_NOTIFICATION")
     private boolean enableEmailNotification;
+
+    @Lob
+    @ElementCollection
+    @CollectionTable(name="PROCESS_EMAIL", joinColumns = @JoinColumn(name="PROCESS_ID"))
+    @Column(name = "EMAIL_ADDRESSES")
     private List<String> emailAddresses;
+
+    @Column(name = "STORAGE_RESOURCE_ID")
     private String storageResourceId;
+
+    @Column(name = "USER_DN")
     private String userDn;
+
+    @Column(name = "GENERATE_CERT")
     private boolean generateCert;
+
+    @Column(name = "EXPERIMENT_DATA_DIR", length = 512)
     private String experimentDataDir;
+
+    @Column(name = "USERNAME")
     private String userName;
 
```

Comment:
Looks like this table is missing USE_USER_CR_PREF.

Reply:
It means to use the user's own UserComputeResourcePreferences, in other words, log into the compute resource using the user's own account instead of the "community" account that gateway users normally used.

It maps to useUserCRPref on the model: https://github.com/apache/airavata/blob/001a6f5192911b2b6e071a83aec09201f05ac1b5/thrift-interface-descriptions/data-models/experiment-catalog-models/process_model.thrift#L66-L66